models/user.model.js

```diff
@@ -9,9 +10,17 @@ const UserSchema = new mongoose.Schema(
             trim: true,
             lowercase: true,
         },
-        password: { type: String, required: [true, 'Password is required'] },
+        password: {
+            type: String,
+            required: [true, 'Password is required'],
+            select: false,
```

Comment:
This select param can produce error somewhere because from the beginning we dont explicitly query password everywhere.